src/services/request.service.js

```diff
@@ -76,12 +76,7 @@ const getOneRequest = async (requestId) => {
       ]
     },
     {
-      model: db.comment,
-      include: [{
-        model: db.user,
-        as: 'author',
```

Comment:
I think @Mastel22 will need this for her comments feature. Ask her if you should omit it.